guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -20,8 +20,16 @@ ifdef::satellite[]
 +
 Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs):: `{RepoRHEL9BaseOS}`
 +
+Ensure this repository is enabled for the latest minor and major versions.
```

Comment:
This is technically correct, but I wonder if the reader will understand what we mean.

Take this screenshot (and ignore the ARM part, I have x86 repos already enabled, so they are not displayed in the list)
![image](https://github.com/user-attachments/assets/f1ed1d83-1641-4608-865f-1afe4755e219)

They need to enable "aarch64 9" (note there is no dot-something!) and "aarch64 9.5"

Wonder if the RHEL docs have a name for those repositories.